draft-ietf-quic-transport.md

```diff
@@ -543,6 +544,17 @@ the server supports, as shown below.
 ~~~
 {: #version-negotiation-format title="Version Negotiation Packet"}
 
+The value in the Unused field is selected randomly by the server. The Connection
+ID field echoes the corresponding value from the triggering client packet.  This
+allows clients some assurance that the server received the packet and that the
+Version Negotiation packet is in fact from the server.  The Version field MUST
+be set to 0x00000000.  The remainder of the Version Negotiation packet is a list
+of 32-bit versions which the server supports.
+
+A Version Negotiation packet is never explicitly acknowledged in an ACK frame by
```

Comment:
It doesn't have a packet number anymore, so how could it be acknowledged?  Maybe change to "cannot be explicitly acknowledged.."